HackerRank - Binary Calculator/js/binaryCalculator.js

```diff
@@ -0,0 +1,86 @@
+let button1 = document.getElementById("btn1");
```

Comment:
I think everything thing here should be a `const`, Remember `const -> let -> var`, and never use `var`